src/containers/MapControls.jsx

```diff
@@ -325,12 +333,54 @@ class SelectorPanel extends Component {
         }});
       });
   }
+  
+  //----------------------------------------------------------------
+
+  prepareSubjectsForAssignments(e) {
+    //First things first: make sure the user sees what she/he is going to download.
+    //this.updateMe(null);  //ERROR: TODO: https://github.com/zooniverse/wildcam-gorongosa-education/issues/229
+
+    this.setState({
+      generalDialog: {
+        status: DialogScreen.DIALOG_ACTIVE,
+        message: 'Preparing data...'
+    }});
+
+    const sqlQuery = this.props.selectorData.calculateSql(config.cartodb.sqlQuerySubjectIDs);
+    fetch(config.cartodb.sqlApi.replace('{SQLQUERY}', encodeURI(sqlQuery)))
+      .then((response) => {
+        if (response.status !== 200) {
+          throw 'Can\'t reach CartoDB API, HTTP response code ' + response.status;
+        }
+        return response.json();
+      })
+      .then((json) => {
+        if (json && json.rows) {
+          console.log(json.rows.map((i) => { return i.location }).join(','));
```

Comment:
The console.log() has just been removed - it's superfluous - but the shortening of the `.map()` function has been successfully integrated.